README.CentOS7.md

```diff
@@ -12,16 +11,17 @@ For further details on the install process, please see the LORIS GitHub Wiki Cen
 
 # System Requirements
 
-Default dependencies installed by CentOS 6.x may not meet the version requirements LORIS deployment or development.
-* MySQL 5.7 is supported for LORIS 18.*
-* PHP 7 is supported for LORIS 18.* - upgrade your PHP manually
+Default dependencies installed by CentOS 7.x may not meet the version requirements LORIS deployment or development.
+* MySQL 5.7 is supported for LORIS 21.*
```

Comment:
should not that MariaDB (10.?) is also supported as it is the default for CentOS 7
@ridz1208 which version of MariaDB is the minimum?

Reply:
Dave says on slack - 10.3